vicare/water_heater.py

```diff
@@ -43,13 +52,13 @@
 }
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
```

Comment:
I belive we were not able to use async because we do I/O when restoring the token.
Did that behavior change in any way?

Reply:
Good point, I haven't thought of that.

Reply:
works nicely unless the token expires :D

Reply:
You're right 👍🏽

Reply:
Just had a more in-depth look, this is only setting up the platform, as there is no API called, this is safe to call async. Actually, `climate.py` is already async.